python/shotgun_model/simple_shotgun_model.py

```diff
@@ -67,10 +70,21 @@ def load_data(self, entity_type, filters=None, fields=None, order=None, limit=No
                   is handling, allowing a user to for example show the twenty most recent notes or
                   similar.
         :param columns: List of Shotgun fields to use to populate the model columns
+        :param additional_filter_presets: List of Shotgun filter presets to apply, e.g.
+                  ``[{"preset_name":"LATEST","latest_by":"BY_PIPELINE_STEP_NUMBER_AND_ENTITIES_CREATED_AT"}]``
```

Comment:
This looks quite magical to me. Are there any type of doc reference or context we can provide with info about what these additional presets mean, where you can find a list of what's available and how they are meant to be used?